js/binance.js

```diff
@@ -3556,7 +3561,11 @@ module.exports = class binance extends Exchange {
             if (since !== undefined) {
                 request['startTime'] = since;
                 // max 3 months range https://github.com/ccxt/ccxt/issues/6495
-                request['endTime'] = this.sum (since, 7776000000);
+                let endTime = this.sum (since, 7776000000);
+                if (until !== undefined) {
+                    endTime = Math.min (endTime, until);
+                }
+                request['endTime'] = endTime;
```

Comment:
Let's only send it if `until` was specified in params (we should not send endTime unconditionally).

Reply:
Actually, nevermind, i guess we have to do it in this specific case.